lib/src/bt_device.dart

```diff
@@ -241,6 +241,42 @@ class GYWBtDevice with ChangeNotifier implements Comparable<GYWBtDevice> {
     this.font = font;
   }
 
+  /// Set the screen brightness. The value must be between 0 and 1.
```

Comment:
You can add an `assert` or trigger an `Exception` or something like this to ensure the value is between 0 and 1.